clappr/clappr-ios
# Placeholder Drawer

## Goal
- To create a way to add an empty place below the mediaControl, in order to not collide with the drawers.

## How to test
1. On `DashboardViewController`, add a new option `kMediaControlBottomPadding: 36`
2. Run the sample app
3. Play the video and tap on the player to show the media control
4. Notice that the mediaControl will have a bottom padding of 10 pixels

![Simulator Screen Shot - iPhone Xʀ - 2019-09-06 at 09 41 32](https://user-images.githubusercontent.com/997521/64428613-8d86d980-d08a-11e9-84a3-b331863d3ef2.png)
